### ISSUE TITLE
scalar values/ annual sums per region

### ISSUE THREAD

**Issue by gplssm** [windnode/WindNODE_ABW#18]
Create another dataframe in results keyed "Jahresmengen nach Gemeinde" which is index by _Gemeinde key_. This dataframes serves simple creation of plot. The following columns are requested:

- [x] Jahresmenge Stromnachfrage [MWh]
- [x] Jahresmenge Wärmenachfrage [MWh]
- [x] Jahresmenge Stromerzeugung (je Technologie) [MWh]
- [ ] ~~Jahresmenge dezentr. PtH [MWh]~~
- [ ] ~~Jahresmenge zentr. PtH [MWh]~~
- [x] Jahresmenge Wärmeerzeugung (je Technologie) [MWh]
- [x] Jahresmenge Leistungsaufnahme Batterie [MWh]
- [x] Jahresmenge Leistungsabgabe Batterie [MWh]
- [x] Jahresmenge Batteriespeicherverluste [MWh]
- [x] Jahresmenge DSM (verschobene Energie) [MWh]
- [x] Jahresnettobilanz Stromaustausch [MWh]
- [x] Jahresmenge Stromexport [MWh]
- [x] Jahresmenge Stromimport [MWh]
- [x] Eigenversorgung bilanziell [%]
- [x] Eigenversorgung pro Zeitschritt [%]
- [x] Flächenbedarf Wind [ha]
- [ ] ~~Flächenbedarf Wind [%]~~ #31
- [x] Flächenbedarf PV Freifläche [ha]
- [ ] ~~Flächenbedarf PV Freifläche [%]~~ #31
- [x] Flächenbedarf PV Aufdach [m²]
- [ ] ~~Flächenbedarf PV Aufdach [%]~~ #31
- [x] CO2 Emission [t CO2] (gerne auch nach Technologie und/oder Sektor aufgelöst und davon abgeleiten), inkl. Netz #32
- [ ] ~~CO2 Emission [% 1990]~~ nicht möglich weil Bezugsgröße fehlt
- [ ] ~~Gesamtkosten in €~~ -> #35 
- [ ] ~~Stromversorgungskosten in €/MWh~~ -> #35 
- [ ] ~~Wärmeversorgungskosten in €/MWh~~ -> #35 
- [ ] ~~Investitionen je Technologie (separat in Spalte), inkl. Netz #32~~ -> #36
- [ ] ~~Netzausbau je Line #32~~ -> #36
- [ ] ~~Netzauslastung nach Netzausbau, relativ je Line~~ -> #36
- [ ] ~~Netzverluste in MWh~~ -> #36

Feel free to add more, if they make sense for analyzing results.

**Comment by nesnoj** [comment]
I added some grid related stuff above...

**Comment by nesnoj** [comment]
Reminder regarding "Eigenversorgung": Differentiate between 1) el. energy only, 2) el.+th. energy

**Comment by nesnoj** [comment]
Reminder: Are there `Link` flows in both directions at the same time?

**Comment by gplssm** [comment]
Moved costs calculation to a separate issue: #35 

Likewise, for ex-post calculations for grid results: #36

**Comment by nesnoj** [comment]
@gplssm We discussed why there're only energy-related emissions for GuD instead of power-related in our tech table, remember?

@nailend and I reviewed this again and we couldn't find any sources for fixed emissions. It does not make much sense to calculate total emissions by using the energy-related ones only as we might encounter just 100 FLH in 2050. Therefore, I propose to convert emis_var->emis_fix beforehand by using typ. FLH range of 3000..4000 given in [ISE](https://www.ise.fraunhofer.de/content/dam/ise/de/documents/publications/studies/DE2018_ISE_Studie_Stromgestehungskosten_Erneuerbare_Energien.pdf)).

I adjusted the table accordingly.
